.github/workflows/wheels.yml

```diff
@@ -0,0 +1,192 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Final releases, release candidates and post-releases build
+      - "[0-9]+.[0-9]+.[0-9]+(rc[0-9]+)?(.post[0-9]+)?"
+  pull_request:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+.X"
+
+jobs:
+  # Check whether to trigger the build
+  check_build:
+    name: Check build trigger
+    runs-on: ubuntu-latest
+    outputs:
+      build: ${{ steps.check_build.outputs.build }}
+
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@v1
+
+      - name: Set environment variables
+        run: |
+          echo "COMMIT_MSG=$(git log --no-merges -1 --oneline)" >> $GITHUB_ENV
+
+      - id: check_build
+        name: Check build
+        if: endsWith(env.COMMIT_MSG, '[cd build]') || github.event_name == 'push'
+        run: |
+          echo "::set-output name=build::true"
+
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }} for Python ${{ matrix.python }}
+    runs-on: ${{ matrix.os }}
+    needs: check_build
+    if: needs.check_build.outputs.build
+
+    strategy:
+      # Avoid that the wheel builder for a
+      # runner is cancelled if other fails
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        # Build the wheels for Python 3.6 and newer
+        python: [36, 37, 38]
```

Comment:
Shall we add Python 3.9?

Reply:
We cannot since scipy did not release python 3.9 wheels yet.